server/src/main/java/com/cloud/deploy/DeploymentPlanningManagerImpl.java

```diff
@@ -754,9 +754,17 @@ public void checkForNonDedicatedResources(VirtualMachineProfile vmProfile, DataC
         //Only when the type is instance VM and not explicitly dedicated.
         if (vm.getType() == VirtualMachine.Type.User && !isExplicit) {
             //add explicitly dedicated resources in avoidList
-
+            if (s_logger.isDebugEnabled()) {
+                s_logger.debug("Adding pods to avoid lists for non-explicit VM deployment: " + allPodsInDc);
+            }
             avoids.addPodList(allPodsInDc);
+            if (s_logger.isDebugEnabled()) {
+                s_logger.debug("adding clusters to avoid lists for non-exlicit vm deployment: " + allClustersInDc);
```

Comment:
in line with the other logs

```suggestion
                s_logger.debug("Adding clusters to avoid lists for non-exlicit VM deployment: " + allClustersInDc);
```